spec.bs

```diff
@@ -7117,7 +7228,15 @@ To <dfn>adjust bid list based on k-anonymity</dfn> given a [=list=] of [=generat
   1. Set |bidCopy|'s [=generated bid/for k-anon auction=] to false.
   1. [=Apply any component ads target to a bid=] given |bidCopy|.
   1. [=list/Append=] |bidCopy| to |bidsToScore|
-  1. If [=query generated bid k-anonymity count=] given |generatedBid| returns true:
+  1. Let |selectedReportingId| be a [=string=]-or-null that is set to null.
+  1. If |generatedBid|'s [=generated bid/selected buyer and seller reporting ID=] [=map/exists=], set 

```

Comment:
|generatedBid| is struct, not a map, so we check if the field's value is null instead. The {{AuctionAd}} is a dictionary (map), so we checked {{AuctionAd/selectableBuyerAndSellerReportingIds}} [=map/exists=] there.
```suggestion
  1. If |generatedBid|'s [=generated bid/selected buyer and seller reporting ID=] is not null, set
```